aptos-move/aptos-transaction-benchmarks/src/transactions.rs

```diff
@@ -218,8 +229,20 @@ where
 
         let state_view = Arc::new(executor.get_state_view().clone());
         let parallel_block_executor =
```

Comment:
I mean, were you arguing this thing returns `VMStatus` so the client must return `VMStatus`? I'd argue this thing is at the same level of the `executor::BlockExecutor` which doesn't return `VMStatus`. I think it's awkward to convert things like your `executor_service::Error` into VMStatus when it's not necessary (okay I see you unwraped on it, but it's not really necessary, right?).

Reply:
Discussed offline, this is one level lower than the `BlockExecutor`. For now, this is okay - we can revisit this infra further once we evolve the design.